webrtc/interfaces.html

```diff
@@ -0,0 +1,118 @@
+<!doctype html>
+<html>
```

Comment:
Much of the introductory markup here can be omitted if you like, everything between `<!doctype html>` and `<script src=/resources/testharness.js></script>` in fact. People have different preferences, I won't nitpick it further, but at the very least not much is *required*. (I like very little boilerplate.)